.github/workflows/coder.yaml

```diff
@@ -64,10 +64,18 @@ jobs:
               - '**'
             docs:
               - 'docs/**'
-              # For testing:
-              # - '.github/**'
             sh:
               - "**.sh"
+            gen:
+              - coderd/database/dump.sql
+              - coderd/database/querier.go
+              - provisionersdk/proto/provisioner.pb.go
+              - provisionerd/proto/provisionerd.pb.go
+              - site/src/api/typesGenerated.ts
```

Comment:
This should have the dependencies for each of these generated files as well.

Reply:
Good point! I'm not familiar with that part of the codebase. What should those be?

Reply:
Ah cool! Didn't realize that, I think I got them all. How's it look now?